ports/geckolib/values.rs

```diff
@@ -81,3 +83,20 @@ pub fn convert_nscolor_to_rgba(color: u32) -> RGBA {
         alpha: (((color >> 24) & 0xff) as f32) / 255.0,
     }
 }
+
+#[inline]
+pub fn round_border_to_pixels(width: Au) -> Au {
+    // Round width down to the nearest pixel, but any non-zero value that would
+    // round down to zero is clamped to 1 pixel.  Used for storing computed
+    // values of border-*-width and outline-width.
+    //
+    // Really we need to round to device pixels, but for that we need to know
+    // the device pixel to app unit ratio, which we don't funnel through from
+    // Gecko.  For now just assume app units per device pixels is the same
+    // as AU_PER_PX (which is app units per CSS pixel).
```

Comment:
Don't we have this already as mTwipsPerPixel?

If that doesn't work for some reason, seems like we should mark this FIXME so that we can easily grep for it later.